node-notes-cli/app.js

```diff
@@ -0,0 +1,52 @@
+import * as fs from 'node:fs/promises';
+
+try {
```

Comment:
This try/catch isn't going to do anything useful because you don't use `await` anywhere. You should `await` each of the functions in this try/block.